src/libraries/System.DirectoryServices/ref/System.DirectoryServices.manual.cs

```diff
@@ -5,6 +5,70 @@
 // ------------------------------------------------------------------------------
 
 using System.DirectoryServices.Design;
+using System.Runtime.CompilerServices;
+
+#if NETSTANDARD2_0
```

Comment:
Let's try to move these types on all frameworks.

Reply:
Done; also updated the main PR description to include the details on the 2 assemblies and forwarding.